pkg/utils/kubeconfig/kubeconfig_test.go

```diff
@@ -23,7 +23,7 @@ func TestCreateNewKubeConfig(t *testing.T) {
 			"test-context": {AuthInfo: "test-user", Cluster: "test-cluster", Namespace: "test-ns"}},
 	}
 
-	err := kubeconfig.WriteToFile(configFile.Name(), &testConfig, false)
+	err := kubeconfig.WriteKubeCfg(configFile.Name(), &testConfig, false)
```

Comment:
Actually, I think we should just call this`kubeconfig.Write` – wdyt?

Reply:
Yes i think you are right. That would be simpler.  I'll make the change.